bin/clang/util.cc

```diff
@@ -283,6 +303,23 @@ Expr* tokenToExpr(DeclContext* ctx, const Token& tok, clang::SourceLocation loc)
   }
 }
 
+Expr* zeroExpr(clang::SourceLocation loc)
+{
+  if (CompilerGlobals::astContextLists.enclosingFunctionDecls.empty()){
+    internalError(loc, "no active DeclContext - cannot make zeroExpr");
+  }
+
+  FunctionDecl* fd = CompilerGlobals::astContextLists.enclosingFunctionDecls.back();
+  if (!fd){
```

Comment:
In what context do we have a DeclContext, but it is Null?

Reply:
prob never